packages/gatsby-cli/src/create-cli.js

```diff
@@ -288,22 +305,24 @@ function isLocalGatsbySite() {
   return !!inGatsbySite
 }
 
-function getVersionInfo() {
-  const { version } = require(`../package.json`)
+function getVersionInfo(cliVersion) {
+  const nodeVersion = getCurrentNodeVersion() ?? `unknown`
   const isGatsbySite = isLocalGatsbySite()
   if (isGatsbySite) {
     // we need to get the version from node_modules
-    let gatsbyVersion = getLocalGatsbyVersion()
-
-    if (!gatsbyVersion) {
-      gatsbyVersion = `unknown`
-    }
-
-    return `Gatsby CLI version: ${version}
-Gatsby version: ${gatsbyVersion}
-  Note: this is the Gatsby version for the site at: ${process.cwd()}`
+    const gatsbyVersion = getLocalGatsbyVersion() ?? `unknown`
+    const sharpVersion = getLocalSharpVersion() ?? `not installed`
+
+    return `Current call:
+  Gatsby CLI version: ${cliVersion}
+  NodeJS version: ${nodeVersion}
+For the site at: ${process.cwd().replace(/^\/home\/[^/]+/, `~`)}
```

Comment:
this doesn't work for window machines, I know we have some crazy replacements inside the telemetry section.
https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-telemetry/src/error-helpers.js

Reply:
I realized that and at the same time it doesn't hurt windows either. Tbh I wasn't trying to scrub the username, but rather eliminate a common part of the absolute path.

Do you feel this should go the extra mile for windows users? (It's absolutely only used here for printing in the cli)

Reply:
if you don't do it for windows users, why do it for linux/mac users? :)

Reply:
> if you don't do it for windows users, why do it for linux/mac users? :)

Low hanging fruit. For linux it's a straightforward one-liner. For windows it's a whole library.

> We're not sending it to anywhere so why cut it

I for one don't like posting my username. Having it scrubbed like this seemed like a simple enough solution.

Tbh I don't even understand what the linked telemetry cleanup is doing. How is that catching user folders like the home dir? I haven't worked on windows in a long time, I thought it was something like `c:\User Files\Your Name\`? I've seen many variations of that. I'm not sure how that regex scrubs those at all or how you could trivially do so in the first place.

Ultimately, I don't feel strong about this. I don't think it's harmful to keep the `/home/user` -> `~` replacement. If you feel strongly about not doing it because of inconsistencies to Windows then ok, let's drop the `replace` call.

Reply:
IMO you can roll with the replace. It's not harmful or critical in any way, just a nice additionn.